External/SPMmodified/spm_load.m

```diff
@@ -191,3 +225,50 @@
     x = struct2cell(x);
     x = [x{:}];
 end
+
+
+end
+
+%% ========================================================================
+
+function [d] = FixIllegalEmptyCells(d, N)
```

Comment:
Sorry, but that's a very messy function. Best would be to detect entire rows with empty cells and remove a list of rows and not run this iteratively, as this is quite inefficient.